jenkins-master/jobs/scripts/workspace/config.py

```diff
@@ -18,11 +20,13 @@
                 'tier': 3,
             },
             'logs': {
```

Comment:
Would it help to distinguish items that are used for "text_log_summary" artifact versus "Job Info" artifact on Treeherder? (For reference: https://treeherder.mozilla.org/api/project/mozilla-central/artifact/?job_id=2716806)